data/hai missions.txt

```diff
@@ -1061,3 +1061,93 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Enlisted by Hai [1]"
+	name "Defend <planet>"
+	description "Defeat a pirate raid on <destination>."
+	to offer
+		"combat rating" > 400
+		"random" < 35
+	source "Allhome"
+	npc
+		government "Hai"
+		personality "heroic" staying"
+		fleet "Large Hai" 2
+		fleet "Large Militia"
+	npc evade
+		government "Pirate"
+		personality "heroic" "staying"
+		fleet "Large Northern Pirates" 10
+	on offer
+		conversation
+			`Suddenly you notice a massive rush in the spaceport: "We are under attack! <planet> is under attack by the unfettered humans! We need every combat-worthy ship to join in the defenses!" The authorities will probably be very gracious, but this could also be an easy way to get yourself killed.`
+			choice
+				`	(Join the defense fleet.)`
+				`	(Stay here until the fight is over.)`
+					defer
+			`A mix of Hai and Human pilots have amassed to help repel the unfettered attack. You join them, and take off together...`
+				launch
+	on complete
+		dialog
+			`The government of <planet> pays you <payment> for helping to drive off the attack.`
+		payment 280000
+
+mission "Enlisted by Hai [2]"
+	name "Enlist with the Hai"
+	description "You have been recommended for your attentiveness for keeping the peace throughout Hai space. Hai officials would like to consult you about possible enlistment."
+	priority
+	to offer
+		has "Enlisted by Hai [1]: done"
+	to complete
+		never
+	to fail
+		has "enlisted"
+	source
+		government "Hai"
+	destination "Hai-home"
+	on offer
+		conversation
+			`As soon as you depart your ship you discover an envoy of sorts awaiting you. "Few human visitors demonstrate care for the lasting peace we have established so long ago," the Hai begins hastily. "You are a prominent example of your kind, and we request audience on <destination>. The unfettered humans have revealed a tentativeness to prey on those who follow a path unlike their own."`
+			choice
+				`	"I suppose I can hear you out."`
+				`	"I would prefer to not get caught up in any trouble."`
+					defer
+			`	"That is most pleasing," it says. "the consuls will be notified of this news and will be expecting attendance."`
+				accept
+
+mission "Enlisted by Hai [3]"
+	name "Hai Bounty"
+	description "Eliminate the coordinator of the attacks on Hai space."
+	landing
+	to offer
+		has "Enlisted by Hai [2]: offered"
+	source "Hai-home"
+	npc "kill"
+		government "Pirate"
+		dialog
+			`You have eliminated the pirate leader's ships. Hopefully this sends a warning to any other pirate warlords who meddles in the affairs of the Hai.`
+		personality "disables" "heroic" "nemesis" "plunders" "staying" "target"
```

Comment:
Same here, no quotes on single-word tokens that are reserved words (i.e. you can't just throw any ol' text here and have it be used as a personality).